packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -107,6 +113,8 @@ export function ArticleMetadataPanel({value, onClose, onChange}: ArticleMetadata
     fetchPolicy: 'network-only'
   })
 
+  const slugTooltip = <Tooltip>{t('articleEditor.panels.slugifyTitle')}</Tooltip>
```

Comment:
if you only use something at one place it's better if you directly assign it there instead of saving it to a variable first.